package triggerPackage;

message TRIGGER {
	optional string content = 1;
}
